apps/aragon-fundraising/app/src/screens/Reserves.js

```diff
@@ -221,123 +175,114 @@ export default () => {
     }
   }
 
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+
+  const editMonthlyAllocationButton = <Button icon={<img src={EditIcon} />} label="Edit monthly allocation" onClick={() => setOpened(true)}></Button>
+
   return (
-    <ContentWrapper>
-      <div className="settings">
-        <h1 className="title bold">Edit reserve settings</h1>
-        <div className="settings-content">
-          <div css="margin-right: 4rem;">
-            <div css="display: flex; flex-direction: column; margin-bottom: 1.5rem;">
-              {NotificationLabel('Monthly allocation', hoverTextNotifications[0])}
-              <Text as="p" css="padding-right: 1.5rem; font-weight: bold;">
-                {displayRate} DAI / month
-              </Text>
-            </div>
-            <div css="display: flex; flex-direction: column; margin-bottom: 1.5rem;">
-              {NotificationLabel('Floor', hoverTextNotifications[2])}
-              <Text as="p" css="padding-right: 1.5rem; font-weight: bold;">
-                {displayFloor} DAI
-              </Text>
+    <React.Fragment>
+      <Split
+        primary={
+          <Box padding={layoutName === 'small' ? 2 * GU : 3 * GU}>
+            <div
+              css={`
+                display: grid;
+                grid-column-gap: ${3 * GU}px;
+                grid-template-columns: repeat(${layoutName === 'small' ? '1' : '2'}, 1fr);
+                width: 100%;
+              `}
+            >
+              <div>
+                <h1
+                  css={`
+                    margin-bottom: ${3 * GU}px;
+                    ${textStyle('body1')};
+                  `}
+                >
+                  Edit reserve settings
+                </h1>
+                <ReserveSetting label="Monthly allocation" helpContent={helpContent[0]} value={`${displayRate} DAI / month`} />
+                <ReserveSetting label="Floor" helpContent={helpContent[2]} value={`${displayFloor} DAI`} />
+                <div>{layoutName !== 'small' && editMonthlyAllocationButton}</div>
+              </div>
+              <div>
+                {[[daiSymbol, daiRatio], [antSymbol, antRatio]].map(([symbol, ratio], i) => (
+                  <ReserveSetting
```

Comment:
Can we get these labels to align with the `Monthly allocation` and `Floor` labels?

Reply:
Oops, done! https://github.com/AragonBlack/fundraising/pull/114/commits/65615f5ca66f5e03d872a0b05619d42f7a0f909c